### PR TITLE
ContainerInfra v1: Fix cluster Get

### PR DESCRIPTION
**Pull request by openstacker**
This is a regression issue introduced by[1], which may impact the
cluster Get. The health_status_reason attribute should be a dict
instead of a string.

[1] https://github.com/gophercloud/gophercloud/pull/1910

For #1909

### CODE REVIEW

**Comment by openstacker** [comment]
@jtopjian  I'm sorry this is a regression issue. Please review it as it's blocked the very basic cluster Get function. BTW, when the version 0.12 will be released? Thanks.

**Comment by jtopjian** [comment]
> BTW, when the version 0.12 will be released? 

Probably not until next month.

File: openstack/containerinfra/v1/clusters/results.go [hunk 1]
```diff
@@ -69,37 +69,37 @@ func (r ResizeResult) Extract() (string, error) {
...
+	Status             string                 `json:"status"`
+	StatusReason       string                 `json:"status_reason"`
+	UUID               string                 `json:"uuid"`
+	UpdatedAt          time.Time              `json:"updated_at"`
+	UserID             string                 `json:"user_id"`
+	FloatingIPEnabled  bool                   `json:"floating_ip_enabled"`
+	FixedNetwork       string                 `json:"fixed_network"`
+	FixedSubnet        string                 `json:"fixed_subnet"`
+	HealthStatus       string                 `json:"health_status"`
+	HealthStatusReason map[string]interface{} `json:"health_status_reason"`
```
Comment (jtopjian) [inline openstack/containerinfra/v1/clusters/results.go:102]: https://github.com/openstack/magnum/blob/master/magnum/api/controllers/v1/cluster.py#L145

:+1:

**Comment by jtopjian** [comment]
@openstacker We've had a few other small bug fixes come in. If you can confirm this works, I can release version 0.11.1 for you - if that would help?

**Comment by openstacker** [comment]
> @openstacker We've had a few other small bug fixes come in. If you can confirm this works, I can release version 0.11.1 for you - if that would help?

Thank you very much. I think I can just use the latest commit in CPO. That's OK. Thanks again.

**Pull request merged**